delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/ElasticSearchView.scala

```diff
@@ -106,12 +100,9 @@ object ElasticSearchView {
       id: Iri,
       project: ProjectRef,
       uuid: UUID,
-      resourceSchemas: Set[Iri],
-      resourceTypes: Set[Iri],
+      pipeline: List[PipeDef],
       resourceTag: Option[TagLabel],
-      sourceAsText: Boolean,
-      includeMetadata: Boolean,
-      includeDeprecated: Boolean,
+      context: Option[ContextObject],
```

Comment:
I also added an optional context to compress correctly the graph when the pipeline is manipulating the data graph